teamcity-rest-client-impl/src/main/kotlin/org/jetbrains/teamcity/rest/implementation.kt

```diff
@@ -1312,7 +1348,7 @@ private class BuildImpl(bean: BuildBean,
     override val changes: List<Change>
         get() = instance.service.changes(
                 "build:$idString",
-                "change(id,version,username,user,date,comment,vcsRootInstance)")
+                "change(id,version,username,user,date,comment,vcsRootInstance,files)")
```

Comment:
BTW how this will affect performance? I suppose including information about changed files may greatly increase response size. Can we fetch it lazily only when `Change::files` is called?

Reply:
Made it `lazy`